cli/code_scanner.py

```diff
@@ -244,8 +251,11 @@ def scan_documents(context: click.Context, documents_to_scan: List[Document], is
     zipped_documents = InMemoryZip()
 
     try:
+        logger.info("Preparing local files")
```

Comment:
debug

Reply:
I put it on info since I would like to update user on processing

Reply:
Fixed